session_redis/README.rst

```diff
@@ -13,6 +13,8 @@ The storage of sessions in Redis is activated using environment variables.
 * ``ODOO_SESSION_REDIS_PORT`` is the redis port (default is ``6379``)
 * ``ODOO_SESSION_REDIS_PASSWORD`` is the password for the AUTH command
   (optional)
+* ``ODOO_SESSION_REDIS_URL`` is an alternative way to define the Redis server
+  address. It's the preferred way when you're using the ``rediss://`` protocol.
```

Comment:
```suggestion
  address. It's the preferred way when you're using the ``redis://`` protocol.
```

Reply:
`rediss` wasn't a typo. The reason why I added the option to define the address in the form of an URL is because you can't connect to a TLS secured Redis with just the host and port.

Reply:
Ok, I wasn't aware of this!